### PR TITLE
ztp: CNF-9975: Update readmes

### PR DESCRIPTION
**Pull request by irinamihai**
[CNF-9975](https://issues.redhat.com/browse/CNF-9975) - Verify and update readme provided in ZTP container

### CODE REVIEW

File: ztp/gitops-subscriptions/argocd/ImageRegistry.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
 Installation
 -
-1. Use SiteConfig to generate MachineConfig for disk partitioning. Make sure to modify values in the mc appropriately as it is dependent on the underlying disk.   
+1. Use SiteConfig to generate a MachineConfig for disk partitioning. Make sure to modify the values in the MC appropriately as it is dependent on the underlying disk.
```
Comment (irinamihai) [inline ztp/gitops-subscriptions/argocd/ImageRegistry.md:3]: @pixelsoccupied, please review this file when you get the time.
Thanks!

Comment (pixelsoccupied) [reply ztp/gitops-subscriptions/argocd/ImageRegistry.md:3]: /lgtm

**Review by lack — APPROVED** [review]

**Pull request merged**